templates/workspace_services/guacamole/guacamole-server/docker/guacamole/guacamole.properties

```diff
@@ -5,3 +5,8 @@
 
 enable-environment-properties:true
 enable-websocket:true
+enable-sftp:false
+enable-drive:false
+sftp-disable-upload:true
+disable-download:true
+disable-upload:true
```

Comment:
Download is overridden here - https://github.com/microsoft/AzureTRE/blob/a64d1d7e3d69abf205fb32f42d3e8968b433cf6b/templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/src/main/java/org/apache/guacamole/auth/azuretre/connection/ConnectionService.java#L112

Doesn't hurt to have defaults in this file, but can we add a checkbox for upload too.  Both should default to being disabled.

Reply:
maybe we should remove the option to override it . wdyt?

Reply:
Some customers use that to download files prior to airlock. We should also have the upload blocked/optional - that's been overlooked, but was required until we had the airlock.